backend/tests/test_integration_service.py

```diff
@@ -77,31 +84,35 @@ def test_organisation(self):
             org_only,
         )
 
-        with self.subTest('time machine'):
-            old_time = datetime.date(2015, 1, 1).isoformat()
-            new_time = datetime.date(2017, 1, 1).isoformat()
-
-            with freezegun.freeze_time(new_time, tz_offset=1):
-                self.assertRequestResponse(
-                    '/service/o/?at=' + old_time, [],
-                )
-
-                self.assertRequestFails(
-                    '/service/o/456362c4-0ee4-4e5e-a72c-751239745e62/?at=' +
-                    old_time,
-                    404,
-                )
-
-            with freezegun.freeze_time(old_time, tz_offset=1):
-                self.assertRequestResponse(
-                    '/service/o/?at=' + new_time, org_list,
-                )
-
-                self.assertRequestResponse(
-                    '/service/o/456362c4-0ee4-4e5e-a72c-751239745e62/?at=' +
-                    new_time,
-                    org_only,
-                )
+        # The concept of an organisational time span has been
```

Comment:
Just delete the code rather than commenting it out. If we ever need it again, it's right there in Git.